packages/eas-cli/src/metadata/apple/tasks/app-version.ts

```diff
@@ -28,17 +28,17 @@ export type AppVersionData = {
 export class AppVersionTask extends AppleTask {
   private options: AppVersionOptions;
 
-  constructor(options: Partial<AppVersionOptions> = {}) {
+  public constructor(options: Partial<AppVersionOptions> = {}) {
     super();
     this.options = {
       platform: options.platform || Platform.IOS,
```

Comment:
This is not a comment only for this line but I've noticed in the metadata code you're using this pattern a lot. I mean `OPTIONAL || FALLBACK`. I think we should use the nullish coalescing instead.

Reply:
Fixed the 3 instances